sunpy/map/mapbase.py

```diff
@@ -175,6 +136,13 @@ class GenericMap(NDData):
         rest will be discarded.
     """
 
+    _registry = dict()
+
+    def __init_subclass__(cls, **kwargs):
```

Comment:
Given this is pretty arcane and also a very new Python feature a good docstring describing what this is doing in general and for us is probably needed.

Reply:
Fixed. ON THE LINE BELOW.